keyboards/zeal60/rgb_backlight.c

```diff
@@ -1019,15 +1019,15 @@ void backlight_effect_all_off(void)
 void backlight_effect_solid_color(void)
 {
     HSV hsv = { .h = g_config.color_1.h, .s = g_config.color_1.s, .v = g_config.brightness };
-    RGB rgb = hsv_to_rgb( hsv );
+    LED_TYPE rgb = hsv_to_rgb( hsv );
     backlight_set_color_all( rgb.r, rgb.g, rgb.b );
 }
 
 // alphas = color1, mods = color2
 void backlight_effect_alphas_mods(void)
 {
-    RGB rgb1 = hsv_to_rgb( (HSV){ .h = g_config.color_1.h, .s = g_config.color_1.s, .v = g_config.brightness } );
```

Comment:
These should stay as RGB not LED_TYPE... they are never anything other than RGB.

Reply:
reverted